lib/blynk/BlynkLib.py

```diff
@@ -355,6 +360,10 @@ def tweet(self, msg):
     def email(self, to, subject, body):
         if self.state == AUTHENTICATED:
             self._send(self._format_msg(MSG_EMAIL, to, subject, body))
+            
+    def virtualWrite(self, pin, val): # TODO: *args?

```

Comment:
@vshymanskyy why *args? Can you write multiple values at once (a list?) to a virtual pin?